### PR TITLE
healthcare API: formatting fixes and cleanup

### PR DESCRIPTION
**Pull request by noerog**
## Description

Beginning of fixes for:

b/183816408
b/158653137
b/159065286

## Checklist
- [X] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [X] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [X] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [X] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [X] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by noerog** [comment]
@dandhlee What is wrong with the license header?

**Comment by dandhlee** [comment]
the header-check bot unfortunately only recognizes one year format, and not multiple years.
If you reduce from `Copyright 2018, 2019, 2020 Google LLC All Rights Reserved.` to just `Copyright 2018 Google LLC All Rights Reserved.` it should make the header-check bot happy.

**Review by dandhlee — CHANGES_REQUESTED** [review]

File: healthcare/api-client/v1/datasets/datasets_test.py [hunk 1]
```diff
@@ -22,13 +22,12 @@
 import datasets
```
[the remark this answers is not in the document]
Comment (noerog) [reply healthcare/api-client/v1/datasets/datasets_test.py:22]: Done.

File: healthcare/api-client/v1/datasets/README.rst [hunk 2]
```diff
@@ -86,23 +86,39 @@ To run this sample:
...
       {create-dataset,delete-dataset,get-dataset,list-datasets,patch-dataset,get_iam_policy,set_iam_policy,deidentify-dataset}
-        create-dataset      Creates a dataset.
-        delete-dataset      Deletes a dataset.
-        get-dataset         Gets any metadata associated with a dataset.
-        list-datasets       Lists the datasets in the project.
-        patch-dataset       Updates dataset metadata.
-        get_iam_policy      Gets the IAM policy for the specified dataset.
+        create-dataset      Creates a dataset. See
+                            https://github.com/GoogleCloudPlatform/python-docs-
+                            samples/healthcare/api-client/v1/datasets before
```
[the remark this answers is not in the document]
Comment (noerog) [reply healthcare/api-client/v1/datasets/README.rst]: Done.

**Review by dandhlee — APPROVED** [review]
LGTM.

**Comment by noerog** [comment]
Ping @tmatsuo

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**